ceph.spec.in

```diff
@@ -37,16 +37,16 @@
 %{!?_selinux_policy_version: %global _selinux_policy_version %(sed -e 's,.*selinux-policy-\\([^/]*\\)/.*,\\1,' /usr/share/selinux/devel/policyhelp 2>/dev/null || echo 0.0.0)}
 
 %define relabel_files() \
-restorecon -R /usr/bin/ceph-mon > /dev/null 2>&1; \
-restorecon -R /usr/bin/ceph-osd > /dev/null 2>&1; \
-restorecon -R /usr/bin/ceph-mds > /dev/null 2>&1; \
-restorecon -R /usr/bin/radosgw > /dev/null 2>&1; \
+restorecon -R %{_bindir}/ceph-mon > /dev/null 2>&1; \
+restorecon -R %{_bindir}/ceph-osd > /dev/null 2>&1; \
+restorecon -R %{_bindir}/ceph-mds > /dev/null 2>&1; \
+restorecon -R %{_bindir}/radosgw > /dev/null 2>&1; \
 restorecon -R /etc/rc\.d/init\.d/ceph > /dev/null 2>&1; \
 restorecon -R /etc/rc\.d/init\.d/radosgw > /dev/null 2>&1; \
```

Comment:
Here, we can drop the rc.d/init.d lines as well as they contain the deprecated init scripts.